components/Cards/examples/CardExampleViewController.swift

```diff
@@ -24,6 +24,13 @@ class CardExampleViewController: UIViewController {
   var colorScheme = MDCSemanticColorScheme()
```

Comment:
Why do we need both color/shape/typography schemes and a container scheme?

Reply:
The app is currently only injecting those schemes. In order to populate our container scheme with the injected values we need to continue to support those properties.